src/test/resources/goal-output/freecol-0.11.6/net.sf.freecol.common.model.Player_goal.json

```diff
@@ -8413,7 +8413,7 @@
     "parameters": [],
     "returnTag": {
       "comment": "True if this object is not initialized.",
-      "condition": "(target.isUninitialized()) == false ? result==true",
+      "condition": "true ? result==target.uninitialized",
```

Comment:
If I'm not wrong, disposed is a private field in a superclass, then is not visible.